packages/compartment-mapper/test/bundle.test.js

```diff
@@ -52,6 +52,7 @@ const expectedLog = [
     fromMjs: 'foo',
     zzz: 1,
   }),
+  { widdershins: 'againshins' },
```

Comment:
Could you explain the joke? I don't get it and googling did not help.

Reply:
I’m so glad you asked. It’s nonsense. Widdershins means counter-clockwise and its opposite is usually considered deosil, but widdershins has readily recognizable German cognates and deosil has Irish origins. So, I’m asking myself what the opposite of widdershins would be along a shared etymological path and I’m drawing a blank. But, "again", "against", "gainsay", and some other words are all cognates of "gegen" in German, so widdershins could have a Germanic cousin in `againshins`. I still haven’t figured out an opposite.

Reply:
Linguistic competence is a factor in why naming things is hard, just not in the way I thought.

Reply:
We aren’t allowed to use `"foo"` anymore. This is not better, but it is not `"foo"`.

Reply:
There's a `'foo'` above on line 52 ;)